Solar_System_Ana_Lisa_Sutherland_Octo.rb

```diff
@@ -0,0 +1,94 @@
+class Solar_S
+  attr_accessor :planets
+
+  def initialize (planets)
+    @planets = planets
+  end
+  # puts out list of just the planets in the solar system
+  def summary
+    x = 1
+    planet_sum = []
+    @planets.each do |planet|
+      planet_sum << "#{x}. #{planet.name}"
+      x += 1
+    end
+    return planet_sum
+  end
+  # allows user to pick a planet amongst the solar system
+end
+
+class Planet
+  ## Create reader methods to give a user access to read the instance variables.
+  attr_accessor :name, :color, :order, :year_length, :distance_from_the_sun
+  ### Add an initialize method which takes several arguments
+  def initialize (name, color, order, year_length, distance_from_the_sun)
+    @name = name
+    @color = color
+    @order = order
+    @year_length = year_length
+    @distance_from_the_sun = distance_from_the_sun
+  end
+  # adds the new planet created by user to the whole solar system array
+  def planet_bio
+    planet_sum = []
+    planet_sum << "Name:#{name}\nColor:#{color}\nOrder In Solar System:#{order}\nYear Length:#{year_length}\nDistance From The Sun:#{@distance_from_the_sun}"
```

Comment:
Why is `planet_sum` an array?  Why not make it just a `String`?  I don't think you gain anything making it an array.